index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap">
+        <title>Portfolio</title>
+</head>
+
+<body>
+  
+  <header>
+    <nav title="navigation-bar">
+      <button class="nav-button1">
+        <span class="logo">Joy</span>
+      </button>
+  
+      <ul class="ul-nav" title="navigation-list">
+        <li><a class="navigationList" href="#portfolio">Portfolio</a></li>
+        <li><a class="navigationList" href="#about">About</a></li>
+        <li><a class="navigationList" href="#contactme">Contact</a></li>
+      </ul>
+    
+      <div class="nav-button2">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+
+<!--Page 1-->
+<section class="mobilemain" title="Joy's-Portfolio-Home-page">
+ <article class="headline">
```

Comment:
- [OPTIONAL ] It is not necessary to have only one article inside a section tag as in this case. What do you think about removing the article and leaving all the content inside the `section` tag? 🤔